sentry-opentelemetry/sentry-opentelemetry-agent/build.gradle.kts

```diff
@@ -2,7 +2,7 @@ import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
 
 plugins {
     `java-library`
-    id("com.github.johnrengelman.shadow") version "7.1.2"
+    id("com.gradleup.shadow") version "9.0.0-beta8"
```

Comment:
@adinauer I've looked at the shadowed jar locally and it was looking good to me, but would be nice to have your eyes on it as well.

Reply:
I can check, is there a non beta version of it?

Reply:
should be working in https://github.com/getsentry/sentry-java/pull/4157/commits/a99e242f655dfdc3ce81abc3ff7dafb7d6a1303f